xml/System.Text.Json.Serialization.Metadata/JsonPropertyInfo.xml

```diff
@@ -85,9 +106,24 @@ This API is intended for use by the output of the System.Text.Json source genera
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets the JSON property name used when serializing the property.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The value of <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo.Name> cannot conflict with that of other <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo> defined in the declaring <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo.JsonTypeInfo>.
```

Comment:
Seems this type is not nested anymore (see the JsonTypeInfo.xml file).
cc @eiriktsarpalis @layomia 
```suggestion
The value of <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo.Name> cannot conflict with that of other <xref:System.Text.Json.Serialization.Metadata.JsonPropertyInfo> defined in the declaring <xref:System.Text.Json.Serialization.Metadata.JsonTypeInfo>.
```